cdhweb/events/models.py

```diff
@@ -396,6 +396,8 @@ def ical_event(self):
 class EventsLinkPageArchived(LinkPage):
     """Container page that defines where Event pages can be created."""
 
+    Page.is_creatable = False
```

Comment:
We don't need to set this on `Page`, just on "this" class
```suggestion
    is_creatable = False
```
No idea if anything would come from setting it on `Page`, but it seems a little scary